websocket/app/src/handler.rs

```diff
@@ -114,8 +124,10 @@ async fn handle_message(
                 return Ok(None);
             };
 
-            let rooms = state.rooms.try_lock().unwrap();
-            let room = rooms.get(room_id).unwrap();
+            let rooms = state.rooms.try_lock()?;
+            let room = rooms
+                .get(room_id)
+                .ok_or_else(|| WsError::RoomNotFound(room_id.to_string()))?;
             let doc = room.get_doc();
             match yrs::sync::Message::decode_v1(&d) {
```

Comment:
Can you update this part as well to use v2?

For ref:
[decode_v2](https://docs.rs/yrs/latest/yrs/updates/decoder/trait.Decode.html#method.decode_v2)
[encode_state_as_update_v2](https://docs.rs/yrs/latest/yrs/trait.ReadTxn.html#method.encode_state_as_update_v2)